### PR TITLE
Fix time zone issue with call list

### PR DESCRIPTION
**Pull request by jatsqi**
* We no longer compare UTC timestamp with zoned midnight timestamp

### CODE REVIEW

File: src/main/kotlin/com/chillibits/coronaaid/shared/ModelToDtoMappers.kt [hunk 1]
```diff
@@ -4,12 +4,17 @@ import com.chillibits.coronaaid.model.db.*
...
 import java.time.Instant
 import java.time.LocalDate
+import java.time.ZoneId
 import java.time.temporal.ChronoUnit
 
 fun LocalDate.yearsBetween(other : LocalDate): Long =
         ChronoUnit.YEARS.between(this, other)
 fun Instant.truncateToMidnight(): Long =
         this.truncatedTo(ChronoUnit.DAYS).toEpochMilli()
+fun Instant.truncateToLocalMidnight(): Long =
```
Comment (marcauberer) [inline src/main/kotlin/com/chillibits/coronaaid/shared/ModelToDtoMappers.kt:14]: Could we refactor all methods without any context to model mapping into a seperate extensions file?

Comment (jatsqi) [reply src/main/kotlin/com/chillibits/coronaaid/shared/ModelToDtoMappers.kt:14]: Yeah we could move these methods to a seperate file. But in the end the model mapper is the only file where these methods are used. (except the InfectedCompressed DTO)

Comment (marcauberer) [reply src/main/kotlin/com/chillibits/coronaaid/shared/ModelToDtoMappers.kt]: Would move them for better readablility and code separation. As they are extension methods, we would have no disadvantaged overhead due to the extraction.

Comment (marcauberer) [reply src/main/kotlin/com/chillibits/coronaaid/shared/ModelToDtoMappers.kt]: Thanks for resolving.

**Pull request merged**